tests/util.test.ts

```diff
@@ -85,6 +85,16 @@ describe('Util', () => {
         input: 'FOO=bar,ZIP=zap',
         expected: { FOO: 'bar', ZIP: 'zap' },
       },
+      {
+        name: 'value with equals',
+        input: 'FOO=bar=baz',
+        expected: { FOO: 'bar=baz' },
+      },
+      {
+        name: 'value with many equals',
+        input: 'FOO=bar=baz;zip=zap;zinc=atom',
```

Comment:
Can we add a test for comma separated + multiple equals? Something like `FOO=bar=baz,zip=zap=zinc` should work but just to catch any regressions in the future.

Reply:
Done